unicoremc/models.py

```diff
@@ -586,8 +577,25 @@ def get_marathon_app_data(self):
             "labels": {
                 "domain": domain,
                 "country": self.get_country_display(),
-                "project_type": self.application_type.project_type,
-                "staticfiles_path": self.get_staticfiles_path(),
+                "project_type": self.application_type.project_type
+            },
+            "container": {
+                "type": "DOCKER",
+                "docker": {
+                    "image": "universalcore/%s"
+                        % self.application_type.get_qualified_name(),
+                    "forcePullImage": True,
+                    "network": "BRIDGE",
+                    "portMappings": [{"containerPort": 8080, "hostPort": 0}],
```

Comment:
Is this 8080 because it's supposed to be the same as the marathon host port? Perhaps this should be a variable (my docker foo is weak).